Sources/Fuzzilli/Compiler/Compiler.swift

```diff
@@ -57,6 +57,9 @@ public class JavaScriptCompiler {
     /// The next free FuzzIL variable.
     private var nextVariable = 0
 
+    /// Context analyzer to track the context of the code being compiled. Used to distinguish switch and loop breaks.
```

Comment:
Nit: maybe `Used for example to distinguish switch and loop breaks`. Just to make it less likely that the comment will become outdated after future changes